web/components/njwds-extended/VerticalStepIndicator.tsx

```diff
@@ -3,10 +3,12 @@ import { useMountEffect, useOnWindowResize } from "@/lib/utils/helpers";
 
 interface Props {
   number: number;
+  order: number;
```

Comment:
I (now) see the purpose of this prop re resetting the numbering for each section, but I think the naming is a little confusing to me? We're not using this prop to order anything. It seems that it represents the way that the step number displays to the user? Maybe we could rename for clarity?